src/components/UploadForm.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+const UploadForm=()=>{
+    const[selectedFile, setSelectedFile] = ("");
+    const onSubmitHandler = (e) => { 
+        e.preventDefault()
+    }
+    const onChangeHandler = (e) =>{
+        setSelectedFile(e.target.files[0])
+        console.log(e.target.files[0])
+      }
+return(
+    <div className= "container">
+        <form className = "form"
+        onSubmit = {onSubmitHandler}>
```

Comment:
Optional: these can be inlined: `onSubmit={(e) => e.preventDefault()}`

I personally prefer this and think it's easier to read, but there's not wide consensus on that, so more just fyi/up to you which one you think is easier to read.